File: Frontend/2. feladatsor FRONTEND/Frontend/src/app/journey.ts

export interface Journey {
  id: number,
  vehicle: Vehicle,
  country: string,
  description: string,
  departure: string,
  capacity: number,
  pictureUrl: string
}

export interface Vehicle {
  id: number,
  type: string
}

export interface JourneyShort {
  id: number,
  destination: string,
}

File: C#/20240410_Journeys DOCKPANEL COMBOBOX/JourneysGUI/Models/Journey.cs
using System;
using System.Collections.Generic;

namespace JourneysGUI.Models;

public partial class Journey
{
    public int Id { get; set; }

    public int Vehicle { get; set; }

    public string Country { get; set; } = null!;

    public string? Description { get; set; }

    public DateOnly Departure { get; set; }

    public int? Capacity { get; set; }

    public virtual Vehicle VehicleNavigation { get; set; } = null!;
}
